docs/docs/how-to/sourcing-data/headless-cms.md

```diff
@@ -20,40 +20,39 @@ We've included setup guides, and links to the integration documentation as well
-| [Agility CMS](https://agilitycms.com/)    | [guide](/docs/sourcing-from-agilitycms/)                                         | [docs](/plugins/@agility/gatsby-source-agilitycms/) | [starter](/starters/agility/agility-gatsby-starter/)                 |
-| [Prepr CMS](https://prepr.io/)            | [guide](https://docs.prepr.io/docs/frontend-integrations/v1/gatsby)              |                                                     |                                                                      |
-| [Forestry](https://forestry.io/)          | [guide](/docs/sourcing-from-forestry/)                                           |                                                     |                                                                      |
-| [Gentics Mesh](https://getmesh.io)        | [guide](/docs/sourcing-from-gentics-mesh)                                        |                                                     |                                                                      |
-| [Seams-CMS](https://seams-cms.com/)       | [guide](/docs/sourcing-from-seams-cms)                                           |                                                     |                                                                      |
-| [Builder.io](https://www.builder.io/)     | [guide](/docs/sourcing-from-builder-io/)                                         | [docs](/plugins/@builder.io/gatsby/)                | [starter](https://github.com/BuilderIO/gatsby-starter-builder)       |
-| [Flotiq](https://flotiq.com/)             | [guide](/docs/sourcing-from-flotiq/)                                             | [docs](/plugins/gatsby-source-flotiq)               | [starter](https://github.com/flotiq/gatsby-starter-blog)             |
-| [Tina](https://tina.io)                   | [guide](https://tina.io/guides/#gatsby)                                          | [docs](https://tina.io/docs/integrations/gatsby/)   |                                                                      |
+| CMS                                       | Guides                                                                           | Plugin Docs                                         | Official Starter                                            |
+| ----------------------------------------- | -------------------------------------------------------------------------------- | --------------------------------------------------- | ----------------------------------------------------------- |
+| [Cosmic](https://cosmicjs.com/)           | [guide](/blog/2018-06-07-build-a-gatsby-blog-using-the-cosmic-js-source-plugin/) | [docs](/plugins/gatsby-source-cosmicjs)             | [starter](https://github.com/cosmicjs/gatsby-blog-cosmicjs) |
```

Comment:
While we're updating this page, I'd say we should remove outdated guides like linking to a blogpost from 2018 and instead link docs that CMS own: https://docs.cosmicjs.com/guides/gatsby

If we don't have our own guide and the CMS has a guide, let's link it!

Reply:
Sounds good. My only concern here is putting Gatsby in the "passenger seat" rather than the "drivers seat" within guides that we link to from here, but I suppose that's not entirely bad.